tests/handles.test.ts

```diff
@@ -0,0 +1,126 @@
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('Set User Handle', () => {
+  test('Logged In User Can Successfully Set Their Own Handle', async () => {
+    // Setup Required Data
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+
+    // Persist to Database
+    await new PortalState().createUsers(member).write();
+
+    // Get API Controller
+    const userController = ControllerFactory.user(conn);
+    const handle = 'mycustomhandle';
+
+    // Get Updated User Profile
+    const patchedUserResponse = await userController.patchCurrentUser({ user: { handle } }, member);
```

Comment:
nit: for consistency with other tests, let's remove the comments here since all tests follow this structure, and let's make the test descriptions start with uppercase and every other word be lowercase instead of all uppercase

Reply:
also nit: 'Logged In User Can Successfully Set Their Own Handle' -> 'Logged-in user can *update* their own handle' so that it's clear that what's being tested is the update rather than the "set" which can happen in registerUser